pkg/subvolume/subvolume.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subvolume
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	exec "github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+)
+
+var getpvlist = `
+kubectl get pv -o jsonpath={.items[*].spec.csi.volumeAttributes.subvolumeName}`
```

Comment:
This is getting all PVs of all types in the cluster, not just rbd/cephfs PVs? Or does the `subvolumeName` in the jsonpath effectively help us filter out all others?

Reply:
@subhamkrai In a separate PR, we need to factor out so `kubectl` can be replaced by another tool such as `oc`.

Reply:
> This is getting all PVs of all types in the cluster, not just rbd/cephfs PVs? Or does the `subvolumeName` in the jsonpath effectively help us filter out all others?

no it will not list all pvs, if you see the subvolumename is under csi, hence it will only list down the cephfs/rbd pvs.